docker-compose.yml

```diff
@@ -52,11 +58,16 @@ services:
   hatchet-engine:
     image: ghcr.io/hatchet-dev/hatchet/hatchet-engine:v0.3.0-alpha.0
     command: /hatchet/hatchet-engine --config /hatchet/config
+    restart: on-failure
     depends_on:
       setup-config:
         condition: service_completed_successfully
       migration:
         condition: service_completed_successfully
+      rabbitmq:
```

Comment:
I suppose we should add the same conditions to `hatchet-migrate` and `setup-config`?

Reply:
done